src/components/KymaModules/components/CommunityModulesList.tsx

```diff
@@ -77,18 +79,25 @@ export const CommunityModulesList = ({
   const handleShowAddModule = () => {
     setLayoutColumn({
       startColumn: {
-        resourceType: 'kymamodules',
+        resourceType: 'kymas',
+        rawResourceTypeName: 'Kyma',
+        namespaceId: 'kyma-system',
+        apiGroup: 'operator.kyma-project.io',
+        apiVersion: 'v1beta2',
```

Comment:
Do we need such specificity? In case of a Kyma update, we need to remember to change it.